index.html

```diff
@@ -8,10 +8,160 @@
+            <a href="#badges"><li>Badge</li></a>
+            
+        </ul>
+    </nav>
+
+    <main>
+
+        <section id="avatar">
+            <h3>Avatar</h3>
+            <div class="demo">
+                        <img src="https://images.all-free-download.com/images/graphicthumb/sable_german_shepherd_514168.jpg" alt="image" class="avatar avatar-sm">
+                        <img src="https://images.all-free-download.com/images/graphicthumb/sable_german_shepherd_514168.jpg" alt="image" class="avatar avatar-md">
+                        <img src="https://images.all-free-download.com/images/graphicthumb/sable_german_shepherd_514168.jpg" alt="image" class="avatar avatar-lr">
+                        <img src="https://images.all-free-download.com/images/graphicthumb/sable_german_shepherd_514168.jpg" alt="image" class="avatar avatar-xl">
+            </div>
+        </section>
+
+        <section id="alerts">
+            <h3>Alerts.</h3>
+            <div class="demo">
+            <div class="alert alert-sucess">Sucess alert</div>
+            <div class="alert alert-failure">Failure alert</div>
+            <div class="alert alert-notification">notification alert</div>
+            </div>
+        </section>
+
+        <section id="buttons">
+            <h3>Buttons</h3>
+            <div class="demo">
+            <button class="btn btn-primary">primary</button>
+            <button class="btn btn-secondary">secondary</button>
+            <button class="btn btn-rounded-primary">Rounded primary</button>
+            <button class="btn btn-rounded-secondary">Rounded secondary</button>
+            </div>
+        </section>
+
+        <section id="cards">
+            <h3>Cards</h3>
+
+            <h4>simple card</h4>
+            <div class="demo">
+            <div class="card">
+                <div class="card-body">
+                        <div class="card-title">Title</div>
+                          <p class="card-text">
+                              Lorem ipsum dolor sit amet consectetur adipisicing elit.         
+                              Sunt officia eos quidem, quaerat doloremque veritatis
+                              minus nihil alias eum? Assumenda eum, esse facilis
+                              asperiores qui ratione placeat et omnis error.
+                          </p>
+                </div>
+                <button class="btn btn-rounded-secondary">LEARN MORE</button>
+            </div>
+            </div>
+
+            <h4>Image card</h4>
+            <div class="demo">
+                <div class="card">
+                    <img src="https://via.placeholder.com/250x220" alt="image" />
+                    <div class="card-body">
+                    <div class="card-title">Title</div>
+                    </div>
+                    <button class="btn btn-rounded-secondary">LEARN MORE</button>
+                </div>
+                </div>
+        </section>
+
+        <section id="Typography">
```

Comment:
ids should start with lowercase letters everywhere

just the best approach rest one can use the way he she likes